contracts/Staking.sol

```diff
@@ -352,7 +366,19 @@ contract Staking is Ownable {
         uint256 burnedDUTokens = _amount.mul(p0).div(t0);
         duToken.burnFrom(msg.sender, burnedDUTokens);
         usersPool[_tokenAddress] = usersPool[_tokenAddress].sub(_amount);
-        ERC20(_tokenAddress).safeTransfer(msg.sender, _amount);
+        uint restAmount = _distributeClaimFees(_amount, _tokenAddress);
+        ERC20(_tokenAddress).safeTransfer(msg.sender, restAmount);
+    }
+
+    function _distributeClaimFees(
```

Comment:
Change to `_distributeClaimingRewards` following the `distributeVerificationRewards` convention for incentives in the protocol.